sros2/sros2/verb/create_key.py

```diff
@@ -26,10 +26,16 @@ class CreateKeyVerb(VerbExtension):
     """Create key."""
 
     def add_arguments(self, parser, cli_name):
-        arg = parser.add_argument('ROOT', help='root path of keystore')
+        arg = parser.add_argument(
+            '-k', '--keystore-root-path',
+            help='root path of keystore')
         arg.completer = DirectoriesCompleter()
-        parser.add_argument('NAME', help='key name, aka ROS node name')
+        parser.add_argument(
+            '-c', '--security-contexts', nargs='*', default=[],
```

Comment:
this changes the semantic of the verb.
Before this was creating a single key, now it is taking a list of security context paths and creating them all.

Either behavior should be kept or the name updated to match the new behavior (the latter sounds good to me)

Reply:
Addressed in https://github.com/ros2/sros2/pull/177/commits/103e18c1366ed793c6269f2d9adab8a16aaf7046.